src/Sulu/Bundle/PreviewBundle/Infrastructure/Doctrine/Repository/PreviewLinkRepository.php

```diff
@@ -39,10 +39,10 @@ public function create(string $resourceKey, string $resourceId, string $locale,
     {
         $token = $this->generateToken();
 
-        /** @var class-string<PreviewLinkInterface> $className */
+        /** @var class-string<PreviewLinkInterface>|PreviewLinkInterface $className */
```

Comment:
This is a false indicator as it says its an object and not a string.

I could also not find why it is required when trying to report here something back to phpstan: https://phpstan.org/r/9095f567-476d-4834-951e-a6fe666a7584

Reply:
```suggestion
        /** @var class-string<PreviewLinkInterface>$className */
```